zipline/gens/brokers/ib_broker.py

```diff
@@ -1022,7 +1022,7 @@ def get_realtime_bars(self, assets, frequency):
             ohlcv['volume'] = trade_sizes.resample(resample_freq).sum()
 
             # Add asset as level 0 column; ohlcv will be used as level 1 cols
-            ohlcv.columns = pd.MultiIndex.from_product([[asset, ],
+            ohlcv.columns = pd.MultiIndex.from_product([[symbol, ],
```

Comment:
Hi,
did you see any issues with that? because as I remember it - the history dataframe contains assets not symbols.

Reply:
yes, it was crashing because it tried to access the array with prices[asset.symbol] on the other side. With this change it's running smoothly. I would love to discuss in detail if you like, is there any way to contact you? Maybe discord?

Reply:
yes, we use slack: https://join.slack.com/t/zipline-live/shared_invite/zt-2eswh6cg-RkkAfook7cvREnQfit6qGA